server/models/users.py

```diff
@@ -103,6 +109,8 @@ class UserSchema(Schema):
     id = fields.Int(dump_only=True)
     email = fields.Email(required=True, validate=must_not_be_blank)
     password = fields.Str(required=True, validate=pw_length)
+    firstName = fields.Str(required=True)
```

Comment:
Should add `validate=must_not_be_blank` for firstName and lastName since we want to make sure ppl complete these fields at login